rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -538,6 +526,11 @@ public void onClick(MenuItemClickEvent event) {
             }
         }
 
+        private void disableNodeSourceDeploymentItems(MenuItem deployItem, MenuItem undeployItem) {
+            deployItem.setEnabled(false);
+            undeployItem.setEnabled(false);
+        }
+
         private void init() {
             setSize("16px", "16px");
             getElement().getStyle().setBorderColor("white");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "white" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)